src/views/Applications/ApplicationForm.vue

```diff
@@ -50,7 +69,7 @@
             />
           </div>
           <div
-            v-else
+            v-if="(!appRegV2Enabled && !isDcr) || (appRegV2Enabled && !appIsDcr)"
```

Comment:
Curious why `v-if` and not `v-else-if`? Should this render regardless of the prior `v-if`?

Reply:
There was actually a bug here! for self managed oidc we should show redirect_url and reference Id